hotels.html

```diff
@@ -23,21 +23,27 @@ <h2>Описание гостиниц</h2>
         <ul>
           <li>
             <h3>Amara Resort&Spa</h3>
+            <img src="img/Amara.jpg" width="135" height="90" alt="Amara Resort&Spa">
```

Comment:
```suggestion
            <img src="img/amara.jpg" width="135" height="90" alt="Amara Resort&Spa">
```
Согласно замечанию об именах файлов.